people_admin/urls.py

```diff
@@ -57,4 +59,14 @@
         apply_bulk_edits,
         name="apply_bulk_edits",
     ),
+    re_path(
```

Comment:
a small nit, but let's change the URL to not be case-sensitive, as I've seen that treated poorly by various cache layers before.  delta-sets or deltas